src/Context/ApiContext.php

```diff
@@ -290,14 +308,23 @@ public function getSessionToken()
     /**
      * @param string $fileName
      *
-     * @throws BunqException
+     * @throws BunqException When could not save the context to the given location.
```

Comment:
Doesn't "When the context couldn't be saved to the given location" make more sense?